### PR TITLE
feat(awscdk-app): initialize sample code stack names based on project name

### LINKED ISSUE #1708: (awscdk) Use project name as stackId instead of an hardcode my-stack-dev and my-stack-prod
Sometimes I create a new cdk project and forget to change the stackId name, it´s not a big deal but I think a small change will be good to avoid problems in deploy

maybe something like `${this.project.name}-dev` and `${this.project.name}-prod` on code sample

Capturar

### PR DESCRIPTION
**Pull request by marciocadev**
closes #1708 

---
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by Chriscbr — CHANGES_REQUESTED** [review]
Looks good, just one small change

File: src/awscdk/awscdk-app-py.ts [hunk 1]
```diff
@@ -113,36 +113,34 @@ class AppCode extends Component {
   constructor(project: AwsCdkPythonApp, fileName: string, cdkVersion: number) {
     super(project);
 
-    const src = new SourceCode(project, fileName);
+    const src = new TextFile(project, fileName, {
+      readonly: false,
```
Comment (Chriscbr) [inline src/awscdk/awscdk-app-py.ts:117]: Let's add a `readonly` option to SourceCodeOptions instead so this can stay as SourceCode

Comment (marciocadev) [reply src/awscdk/awscdk-app-py.ts:117]: About that, `marker` file are always `readonly` ? Cause I don't know if a marker file can be editable

If that's the case maybe `readonly` flag turns irrelevant and a read only file always have to take the `marker` flag

File: src/awscdk/awscdk-app-py.ts [hunk 2]
```diff
@@ -113,36 +113,34 @@ class AppCode extends Component {
...
     }
-    src.line(`from ${project.moduleName}.my_stack import MyStack`);
+    src.line(`from ${project.moduleName}.main import MyStack`);
     src.line("");
     src.line("# for development, use account/region from cdk cli");
     src.open("dev_env = Environment(");
-    src.line("account=os.getenv('CDK_DEFAULT_ACCOUNT'),");
-    src.line("region=os.getenv('CDK_DEFAULT_REGION'),");
+    src.line("  account=os.getenv('CDK_DEFAULT_ACCOUNT'),");
+    src.line("  region=os.getenv('CDK_DEFAULT_REGION'),");
```
Comment (Chriscbr) [inline src/awscdk/awscdk-app-py.ts:131]: Undo

**Review by Chriscbr — CHANGES_REQUESTED** [review]

**Review by Chriscbr — APPROVED** [review]

**Pull request merged**